docs/src/__examples__/InputGroup/STATES.tsx

```diff
@@ -0,0 +1,229 @@
+              value: "April",
+            },
+            {
+              label: "May",
+              value: "May",
+            },
+            {
+              label: "June",
+              value: "June",
+            },
+            {
+              label: "July",
+              value: "July",
+            },
+            {
+              label: "August",
+              value: "August",
+            },
+            {
+              label: "September",
+              value: "September",
+            },
+            {
+              label: "October",
+              value: "October",
+            },
+            {
+              label: "November",
+              value: "November",
+            },
+            {
+              label: "December",
+              value: "December",
+            },
+          ]}
+        />
+
+        <InputField error="This text is not displayed" placeholder="DD" type="number" />
+        <InputField
+          error="This text is not displayed"
+          placeholder="YYYY"
+          value="2020"
+          type="number"
+        />
+      </InputGroup>
+    );
+  },
+  exampleVariants: [
+    {
+      name: "Error",
+      code: `
+    (
+      <InputGroup
+        flex={["8 8 15em", "1 1 4em", "2 2 6em"]}
+        label="Departure"
+        error="Enter a year that is not in the past"
+      >
+        <Select
+          error="This text is not displayed"
+          placeholder="Month"
+          value="May"
+          options={[
+            {
+              label: "January",
+              value: "January",
+            },
+            {
+              label: "February",
+              value: "February",
+            },
+            {
+              label: "March",
+              value: "March",
+            },
+            {
+              label: "April",
+              value: "April",
+            },
+            {
+              label: "May",
+              value: "May",
+            },
+            {
+              label: "June",
+              value: "June",
+            },
+            {
+              label: "July",
+              value: "July",
+            },
+            {
+              label: "August",
+              value: "August",
+            },
+            {
+              label: "September",
+              value: "September",
+            },
+            {
+              label: "October",
+              value: "October",
+            },
+            {
+              label: "November",
+              value: "November",
+            },
+            {
+              label: "December",
+              value: "December",
+            },
+          ]}
+        />
+
+        <InputField error="This text is not displayed" placeholder="DD" type="number" />
+        <InputField
+          error="This text is not displayed"
+          placeholder="YYYY"
+          value="2020
+          type="number"
+        />
+      </InputGroup>
+    );`,
+    },
+    {
+      name: "Help",
```

Comment:
In this variant the error message remains, I assume it's because it's being merged with the main example, which has `error`, so error wins.

Reply:
nope, it's because i forgot to add `() =>`, it's strings, so it's easy to miss such thing :/